virttest/env_process.py

```diff
@@ -1295,7 +1300,7 @@ def preprocess(test, params, env):
             except Exception as e:
                 logging.error("Unexpected error: '%s'" % str(e))
             if libvirtd_inst is None:
-                libvirtd_inst = utils_libvirtd.Libvirtd()
+                libvirtd_inst = utils_libvirtd.Libvirtd("virtqemud")
```

Comment:
I think here better remove the "virtqemud" to be more clear

Reply:
For modular daemon env, the configuration file is changed from libvirtd.conf to virtqemud.conf, so I add the specific daemon name here.
utils_libvirtd.Libvirtd() with virtqemud will create an object which can restart virtqemud or libvirtd. If there is no "virtqemud", it will restart all Virt*d or libvirtd. Anyway, each of them can work well. let's wait for others' feedback if they have same problem.  @yafu-1 As feature owner, do you have any suggestions?

Reply:
As long evolution solution, utils_libvirtd.Libvirtd module need step away, I don't suggest use  utils_libvirtd.Libvirtd("xx") to create new split daemon, such as virtqemud and virtproxyd.

Reply:
@chunfuwen I can see why you don't like it. But I hope you understand that this solution is not my personal decision, but the result of the team discussion. This is the way we hope to see at this stage. You've added label to request 2 maintainers and also requested outside maintainers, unfortunately, there was few response. Anyway, what I want to say is we've been waiting for enough time, it's time to move on.

Reply:
@Yingshun, as I mentioned before, this is our final evolution version. Therefore involve in more participants to review will help work out optimized solution. 
Moreover, as per feedback from feature team, the real requirement is not clear, so not sure whether it is right time to move it  forward now.